### ISSUE TITLE
Manpages missing

### ISSUE THREAD

**Issue by talwat** [FineFindus/artem#3]
Hi! I noticed that the doc directory is missing, which is where the manpage is supposed to be according to the README. Is this a mistake or is this intentional?

**Comment by FineFindus** [comment]
No, it is not intentional. What OS do you use/ which release did you download?

**Comment by talwat** [comment]
I just cloned the git repository.

**Comment by talwat** [comment]
Ah, Thanks!
